src/Instrument/ClassLoading/CachePathResolver.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Go! AOP framework
+ *
+ * @copyright Copyright 2014, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\Instrument\ClassLoading;
+use Go\Core\AspectKernel;
+
+/**
+ * Class that manages real-code to cached-code paths mapping.
+ * Can be extended to get a more sophisticated real-to-cached code mapping
+ */
+class CachePathResolver
```

Comment:
I'm trying to introduce aop support in an application that is 10+ years old :)
Its filesystem span isn't quite suitable for plain str_replacing of app_dir to cache_dir, and the only directory that contains both application and libs' source code is '/', which is impossible to use with this str_replace